app/_webparts/webparts/sidebar-ctas/script.js

```diff
@@ -0,0 +1,43 @@
+$(document).ready(function () {
+	
+	if($('.humanise-sidebar-ctas').length){
+		var td = '/en-GB/tools/Pages/book-a-test-drive.aspx';
+		var br = '/en-GB/tools/Pages/request-a-brochure.aspx';
+		var lt = '/en-gb/live-tour';
+		var fr = '/en-GB/find-a-retailer/Pages/find-your-nearest-skoda-retailer.aspx';
+		var cc = 'http://cc-cloud.skoda-auto.com/gbr/gbr/en-gb';
+		var fc = '/en-GB/tools/finance-calculator';
+		var bs = '/en-GB/owners/service-and-maintenance/online-service-booking/Pages/default.aspx';
+
```

Comment:
Do we have to hard-code URLs in javascript? Perhaps this is easier for Sharepoint, but would be better if we can avoid it. Data attributes might work better.

Reply:
Sure. Do agree. I've hardcoded urls on PR coming back, but have a js overwrite (as we know links constantly change for the likes of CC when factory change model codes and therefore direct links. The js looks at page URL so hopefully drop in once and never have to worry again!